### ISSUE TITLE
Search in toc or command menu items only matching beginning of items

### ISSUE THREAD

**Issue by CristiOlariu** [ahrm/sioyek#150]
The search on toc and command menu does not do a proper fuzzy(?) search across all words in toc items or command menu items.

It seems like the search word needs to match the beginning of the item, whereas I think it should match any word in any position in the items.

Btw, the fuzzy(?) search works for bookmarks and highlight searches.

**Comment by Pascalio** [comment]
For TOC, it seems to me that the fuzzy search only takes toc items that are _first level_ as _header 1_ for example. There it is perfectly fuzzy.
For the command menu, yes, only the beginnings seem to be matched...

**Comment by ahrm** [comment]
Added fuzzy search to command menu in bb571d01878b33bf53bf72b55b0b9dce394627c9.
Also as @Pascalio mentioned, we already had fuzzy search for table of contents, however, I had recently introduced a huge bug in table of contents search which caused the lower level elements to be shown only when their parents matched the text too. It should be fixed in 50aa04e58a17ad76fda96f608e7da083cfaeee2b. Thanks for bringing that to my attention.
